### PR TITLE
Fix: Fixed scroll position after page transitions

### LINKED ISSUE #5825: Bug: Scroll position isn't always in the correct position
### Description

When in a folder that has folders named with _ at the start or group by options enabled the scroll will not be at the top.

### Steps To Reproduce

1) Create a folder with enough items for scrolling
2) Create a folder with an _ at the start or use group by options
3) Go to previous folder and back
4) See scroll position not at the top

### Expected behavior

Should start at top or directory always

### Files Version

1.9.23.0

### Windows Version

10.0.19043.1151

### Relevant Assets

_No response_

### PR DESCRIPTION
**Pull request by hishitetsu**
**Resolved / Related Issues**
- [X] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Closes #5825

**Validation**
How did you test these changes?
- [X] Did you build the app and test your changes?
- [ ] Did you check for accessibility? You can use Accessibility Insights for this.
- [ ] Did you remove any strings from the en-us resource file?
   - [ ] Did you search the solution to see if the string is still being used? 
- [ ] Did you implement any design changes to an existing feature?
   - [ ] Was this change approved?
- [X] Are there any other steps that were used to validate these changes?
   1. Navigate to the root of Files' GitHub repository.
   2. Navigate to `artifacts` (next to `.github`).
   3. Navigate back.
   4. Scroll position will be top and `.github` will be seen.

### CODE REVIEW

**Comment by yaira2** [comment]
Does this take into account the group headers as well?

**Comment by hishitetsu** [comment]
> Does this take into account the group headers as well?

Good point. We should consider them.

File: src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs [hunk 1]
```diff
@@ -65,17 +65,16 @@ public DetailsLayoutBrowser() : base()
 
 		protected override void ItemManipulationModel_ScrollIntoViewInvoked(object? sender, ListedItem e)
 		{
-			FileList.ScrollIntoView(e);
-			ContentScroller?.ChangeView(null, FileList.Items.IndexOf(e) * Convert.ToInt32(Application.Current.Resources["ListItemHeight"]), null, true); // Scroll to index * item height
+			(FileList.ContainerFromItem(e) as ListViewItem)?.StartBringIntoView();
```
Comment (hishitetsu) [inline src/Files.App/Views/LayoutModes/DetailsLayoutBrowser.xaml.cs:68]: This should work with or without grouping, but sometimes it scrolls as expected and sometimes it does not scroll at all.
I have no idea how to make it work all the time.

**Comment by hishitetsu** [comment]
It seems very difficult to scroll to the selected item immediately after the page transition while taking into account the group headers.

**Pull request closed without merging**